docs/en/setup/service-agent/java-agent/Application-toolkit-tracer.md

```diff
@@ -0,0 +1,213 @@
+# Span creating
```

Comment:
Please consistent titles, current you have
- span creating
- Capture/continues
- Tag and log of spans

We should use consistent names. Such as, 
- Create Span
- Capture and Continuous Snapshot
- ...

Reply:
how about: 
* Create Span
* Inject/Extract Context Carrier
* Capture/Continue Context Snapshot
* Add Span's Tag and Log
* Async prepare/finish
* ActiveSpan
I try to make every title as verb-object phrase.

Reply:
- Create Span
- Inject and Extract Context
- Capture and Continue Context
- Add Span's Tag and Log
- Set the span in async mode
- Access the Active Span